### PR TITLE
Using UUID.randomUuid() instead of scenario.getId()

### PR DESCRIPTION
**Pull request by letsrokk**
[//]: # (
. Thank you so much for sending us a pull request! 
.
. Make sure you have a clear name for your pull request. 
. The name should start with a capital letter and no dot is required in the end of the sentence.
. To link the request with isses use the following notation: (fixes #123, fixes #321\)
.
. An example of good pull request names:
. - Add Russian translation (fixes #123\)
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

### Context
[//]: # (
Describe the problem or feature in addition to a link to the issues
)

Fix for #130
As per discussion in Gitter room, using UUID.randomUuid() instead of scenario.getId() to make sure that results are saved in files with safe filenames

#### Checklist
- [x] [Sign Allure CLA][cla]
- [ ] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java [hunk 1]
```diff
@@ -23,14 +23,9 @@
 import io.qameta.allure.util.ResultsUtils;
 import java.io.ByteArrayInputStream;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
```
Comment (baev) [inline allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java]: do not use wildcard imports

Comment (letsrokk) [reply allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java]: cradle build throws error I don't use it
```[Coupling | ExcessiveImports] io.qameta.allure.cucumberjvm.(AllureCucumberJvm.java:1)
  A high number of imports can indicate a high degree of coupling within an object.
  https://pmd.github.io/pmd-5.5.4/pmd-java/rules/java/coupling.html#ExcessiveImports```

Comment (baev) [reply allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java]: Add `@SuppressWarnings("PMD. ExcessiveImports")` on class

Comment (letsrokk) [reply allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java]: any advice here?

Comment (baev) [reply allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java]: > Add @SuppressWarnings("PMD.ExcessiveImports") on class

**Comment by baev** [comment]
@letsrokk thanks!

**Pull request merged**